types/mongoose-deep-populate/tsconfig.json

```diff
@@ -12,6 +12,11 @@
         "typeRoots": [
             "../"
         ],
+        "paths": {
```

Comment:
Missed one? Can this use v5?

Reply:
It errors out with v5 https://travis-ci.org/Alorel/DefinitelyTyped/builds/331093839